browser/src/layer/marker/TextInput.js

```diff
@@ -1112,6 +1112,27 @@ L.TextInput = L.Layer.extend({
 				this._map.fire('closementionpopup', { 'typingMention': false });
 			}
 		}
+
+        var formulaAutocompletePopup = L.DomUtil.get('formulaautocompletePopup');
```

Comment:
I would like to unify the code with mention popup above - it's copy pasted with different id.
We can use function with id parameter for now.